src/content/docs/release-notes/lambda-extension-release-notes/lambda-extension-2.3.11.mdx

```diff
@@ -0,0 +1,14 @@
+---
+subject: "Lambda-Extension"
+releaseDate: '2024-02-28'
+version: 2.3.11
+---
+
+## Notes
```

Comment:
This note section is a little confusing, are the crashes being fixed or are we notifying customers that they are happening?